Yoast/wordpress-seo
# Update scores in collapsible headers

## Summary

This PR can be summarized in the following changelog entry:

* Updates icons of the collapsible headers to reflect the SEO/readability overall score.

## Relevant technical choices:

* Added the overall SEO/Readability score to the store using actions and reducers that already existed.
* Used a helper function to connect the possible results to the icons and colors that are used for those results.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Do build steps:
   - `yarn install`
   - `grunt build:js && grunt build:css`
* Visit an edit-post page. 
* Check if the icons and colors used for the headers reflect the score of the post for readability and SEO.
   - green happy face for good a good seo/readability result.
   - Yellow neutral face for an ok seo/readability result.
   - red frowning face for a bad seo/readability result.
   - grey orb for other cases.
* Check if the collapsible headers change accordingly when the readability/seo results change for the page. ( by improving the post or making the post worse ).

## Quality assurance


Fixes #10453
